src/Form/Destination/AbstractFormDestinationType.php

```diff
@@ -55,7 +55,7 @@ final public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
             $count = $this->countCreatedItemsForAnswersSet($item);
         }
 
-        return self::createTabEntry(static::getTypeName(), $count);
+        return self::createTabEntry(static::getTypeName(), $count, "", static::getTargetItemtype()::getIcon());
```

Comment:
Do not hesitate to use named arguments.
```suggestion
        return self::createTabEntry(static::getTypeName(), $count, icon: static::getTargetItemtype()::getIcon());
```